### DISCUSSION TITLE
`getPages()` should be sorted by order defined in meta.json

### DISCUSSION THREAD

**Discussion by smblee** [fuma-nama/fumadocs#894] [General]
As title says, `getPages()` doesn't seem to respect the order defined in meta.json. I am using this in `<DocsCategory />` component and need the pages to be ordered properly.

**Comment by fuma-nama** [comment]
It shouldn’t, the meta.json configures page tree. You can find the corresponding page tree node of a folder name with page url, or folder path (in v14)
